packages/formation-react/src/components/MegaMenu/arrow-right-blue.jsx

```diff
@@ -13,3 +13,5 @@ export const ArrowRightBlueSVG = () => (
     />
   </svg>
 );
+
+export default ArrowRightBlueSVG;
```

Comment:
NIT: Was it necessary to change this from a named export to a default export?

Reply:
I was getting an error about this file specifically in one of my [commits](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fveteran-facing-services-tools/detail/try-bump-sharp/30/pipeline) after changing things:

```
error A page component must export a React component for it to be valid. Please make sure this file exports a React component:

/home/jenkins/workspace/ng-services-tools_try-bump-sharp/packages/formation-react/src/components/MegaMenu/arrow-right-blue.jsx
```

After updating the export the above error went away. Also default exports are used by the rest of the components so this helps with consistency.

Reply:
That makes sense. Thanks for the explanation.